fbchat/graphql.py

```diff
@@ -118,6 +118,16 @@ def graphql_to_group(group):
         message_count=group.get('messages_count')
     )
 
+def graphql_to_room(room):
+    return Room(
+        **vars(graphql_to_group(room)),
```

Comment:
This line is failing:
```
Traceback (most recent call last):
  File "tests.py", line 12, in <module>
    from fbchat import Client
  File "./fbchat/__init__.py", line 5, in <module>
    from .client import *
  File "./fbchat/client.py", line 13, in <module>
    from .graphql import *
  File "./fbchat/graphql.py", line 123
    **vars(graphql_to_group(room)),
                                  ^
SyntaxError: invalid syntax
```
In python 2.7 and python 3.4

Reply:
I reverted the commit because of this, but somehow I still closed the PR

Reply:
And I don't know how to re-open it :p 
I've made a new PR with the given fix